MarvelApiClient/MarvelAPIClientConfig.swift

```diff
@@ -8,9 +8,9 @@
 
 import Foundation
 
-struct MarvelApiClientConfig {
+struct MarvelAPIClientConfig {
 
     static let host = "http://gateway.marvel.com/v1/public/"
-    static let defaultHeaders = ["Accept" : "application/json"]
+    static let defaultHeaders = ["Accept" : "application/json", "Content-Type" : "application/json"]
```

Comment:
As discussed outside of this PR we will remove the check from the BothamNetworking.